testplan/runnable/interactive/base.py

```diff
@@ -227,24 +227,35 @@ def reset_test(self, test_uid, await_results=True):
             self._update_reports([(self.test(test_uid).dry_run().report, [])])
 
     def run_all_tests(
-        self, await_results: bool = True
+        self,
+        shallow_report: Optional[Dict] = None,
+        await_results: bool = True,
     ) -> Union[TestReport, Awaitable]:
         """
         Runs all tests.
 
+        :param shallow_report: shallow report entry, optional
         :param await_results: Whether to block until tests are finished,
             defaults to True.
         :return: If await_results is True, returns a testplan report.
             Otherwise, returns a future which will yield a testplan report when
             ready.
         """
         if not await_results:
-            return self._run_async(self.run_all_tests)
-
-        self.logger.debug("Interactive mode: Run all tests")
+            return self._run_async(
+                self.run_all_tests, shallow_report=shallow_report
+            )
 
-        for test_uid in self.all_tests():
-            self.run_test(test_uid)
+        if shallow_report:
+            self.logger.debug("Interactive mode: Run filtered tests")
```

Comment:
Loglines are a welcome addition, but I do not see it for the others. Also I wonder if it is INFO level already or perhaps USER_INFO.